### DISCUSSION TITLE
Auto comment SQL queries to track bad queries

### DISCUSSION THREAD

**Discussion by rspadim** [grafana/grafana#38301] [Dashboards, Panels & Visualizations]
Hello folks!

Could be possible to change grafana (I don't know where) to report user/url/others info at each SQL query as a comment?

for example, my dashboard have one query:

```
SELECT COUNT(*) AS counts, severity FROM important_table GROUP BY severity
```

instead of send this directly to database, change it and send:

```
--- User: rspadim, dashboard 1234, org 1234
SELECT COUNT(*) AS counts, severity FROM important_table GROUP BY severity
```

with this I could track what dashboard and user is doing in my database (slow query log for example and current process list).

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
